gnuplot/src/axes_common.rs

```diff
@@ -1175,6 +1135,12 @@ pub struct Margins
 	pub bottom: Option<f32>,
 }
 
+impl Default for Margins {
```

Comment:
Is this something clippy wanted? I don't see the value of this for a crate-private type if I don't use the `Default::default()`.

Reply:
`warning: you should consider adding a `Default` implementation for `Margins`
    --> src/axes_common.rs:1147:2
     |
1147 | /     pub fn new() -> Self
1148 | |     {
1149 | |         Margins {
1150 | |             left: None,
...    |
1154 | |         }
1155 | |     }
     | |_____^
     |
     = note: `#[warn(clippy::new_without_default)]` on by default
     = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#new_without_default
help: try adding this
     |
1145 + impl Default for Margins {
1146 +     fn default() -> Self {
1147 +         Self::new()
1148 +     }
1149 + }
`